### PR TITLE
Issue #35 - Reply to Victim's or Prosecutor's Statement

### PR DESCRIPTION
**Pull request by kthompson55**
The TextArea component has a max input length. Even at the maximum length, the PDF form version won't overrun the first page's available space for the statement of disagreement.

image

Page 2 has a signature area, so I didn't write an text inputs for that, and Page 3 is a Certificate of Service page that I believe gets filled out at the courthouse.

### CODE REVIEW

File: frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+          dataKey="person.addressStreet"
+          left={farLeft}
+          top="17.46%"
+        />
+        <PositionedString
+          debugKey="addressStreetCityZip"
+          left={farLeft}
+          top="20.86%"
+        >
+          {`${renderData("person.addressCity")} ${renderData(
```
Comment (joeldenning) [inline frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: Might be better to have a comma between the city and state?

File: frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+      <RenderPage url={secondPageUrl}>
+        <PositionedString
+          dataKey="case.disagreementReason"
+          left={disagreementLeft}
+          top="28.0%"
+          style={{
+            width: "71%",
+            overflowWrap: "anywhere",
+            lineHeight: "31px"
```
Comment (joeldenning) [inline frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: Hmm which browsers have you tested this on? I'm a bit worried that the 8.5x11" print preview might display 31px differently in different browsers.

File: frontend/app/forms/reply-to-statement/reply-to-statement.web.component.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        <TextInput dataKey="case.judgeName" label={__("judge full name")} />
+      </Section>
+
+      <Section name="3. Statement response">
+        <TextArea
+          dataKey="case.opposingStatement"
+          label={__("statement made by opposition")}
+        />
+        <TextArea
+          dataKey="case.disagreementReason"
```
Comment (joeldenning) [inline frontend/app/forms/reply-to-statement/reply-to-statement.web.component.js]: Let's clarify this dataKey's meaning
```suggestion
          dataKey="case.opposingStatementDisagreementReason"
```

File: frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js (hunk 2 above)
Comment (kthompson55) [reply frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: I was developing in FireFox. I will check Chrome, Edge, and IE11 and get back to you on this.

Comment (kthompson55) [reply frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: The app doesn't even load in IE11, so I don't have any pictures for it, but here's a comparison of Chrome, Edge, and FireFox:
image

Unfortunately, having a bunch of really long words causes more frequent line breaks, so there's an overflow of text from the first page area. But the text does align properly with 31px as a lineHeight in all browsers.

I don't think many of our users will be rolling their hand over their keyboard like I did just to get a long input, but we might need to account for unexpected line breaks, especially if the user presses "enter" a lot.

Comment (joeldenning) [reply frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: 👍 thanks for doing that. I think that we don't need to worry about overflowing beyond the allotted area. And am happy to see that it works in all browsers.

Comment (joeldenning) [reply frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: (except IE which we don't support)

Comment (kthompson55) [reply frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js]: Are we good to resolve this conversation, then?

**Pull request merged**